### ISSUE TITLE
Unable to Partially Deprecate

### ISSUE THREAD

**Issue by VastConfinement** [delegateas/DeprecationTool#27]
It appears I am only able to check or uncheck a box. It does not provide the option for the filled square to partially deprecate a field.

**Comment by magesoe** [comment]
Partial deprecation is just a status for you to know that you have deprecated the field by simply prefixing it. By clicking fix partial you update the description, searchable and required level of a field.

**Comment by VastConfinement** [comment]
OK, so the tool doesn't make something partially deprecated, it just identifies fields you manually prefixed as zz_ so that you can apply the other updates more easily. Is that correct? 
I had thought we could apply a partial deprecation within the tool and then later complete the deprecation.

**Comment by thehabbos007** [comment]
Yes that is correct. The idea is that the tool is solely for completing deprecations.

I see that there might be a case for marking a field as "soft deprecated" before deprecating it fully, but we assume that can be done by manually prefixing with zz_ :)
